test/Microsoft.AspNetCore.Mvc.IntegrationTests/SimpleTypeModelBinderIntegrationTest.cs

```diff
@@ -538,6 +538,42 @@ public class SimpleTypeModelBinderIntegrationTest
             Assert.Equal(new[] { "line 1", "line 2" }, entry.RawValue);
         }
 
+        [Theory]
+        [MemberData(nameof(PersonStoreData))]
+        public async Task BindParameter_WithBindNever_IsNotBound(Dictionary<string, StringValues> personStore)
```

Comment:
These tests aren't really related to `SimpleTypeModelBinder`- make a new test class